mediaapi/fileutils/fileutils.go

```diff
@@ -112,7 +112,10 @@ func WriteTempFile(reqReader io.Reader, maxFileSizeBytes config.FileSizeBytes, a
 	if err != nil {
 		return
 	}
-	defer (func() { err = tmpFile.Close() })()
+	defer (func() {
+		if err == nil {err = tmpFile.Close()
```

Comment:
Please format your code with `goimports` before committing. You may use the automation script `./scripts/build-test-lint.sh` for this.